src/entities/event/ui/event-card/event-card.module.scss

```
@@ -43,14 +43,10 @@
     }
 
     .about-organization {
-      margin-top: 32px;
-
-      .organized-by {
-        margin-bottom: 0;
-      }
+      margin-top: 26px;
```

Let's use sizes with `8/4px` step instead
```suggestion
      margin-top: 24px;
```

---

Done